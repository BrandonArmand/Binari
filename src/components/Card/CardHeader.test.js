import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import CardHeader from "./CardHeader";

describe("CardHeader", () => {
    it("renders snapshot", () => {
        const { container } = render(<CardHeader />);
        expect(container).toMatchSnapshot();
    });
    it("renders CardHeader", () => {
        render(<CardHeader>CardHeader</CardHeader>);
        expect(screen.getByText("CardHeader")).toBeInTheDocument();
    });
});
