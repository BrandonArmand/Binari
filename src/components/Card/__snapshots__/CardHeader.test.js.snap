// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader renders snapshot 1`] = `
<div>
  <div
    class="makeStyles-cardHeader-1"
  />
</div>
`;
